test/integration/query-interface.test.js

```diff
@@ -336,7 +336,8 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     it('rename a simple column', function() {
       const self = this;
       const Users = self.sequelize.define('_Users', {
-        username: DataTypes.STRING
+        username: DataTypes.STRING,
+        commit: DataTypes.STRING
```

Comment:
Could you add a comment for that? It's not obvious and might get changed by accident.
Also, wouldn't this be better caught by an SQL generation unit test?